src/xrpld/overlay/detail/PeerImp.cpp

```diff
@@ -2866,6 +2866,9 @@ PeerImp::checkTransaction(
             (stx->getFieldU32(sfLastLedgerSequence) <
              app_.getLedgerMaster().getValidLedgerIndex()))
         {
+            JLOG(p_journal_.warn())
+                << "Marking transaction " << stx->getTransactionID()
+                << "as BAD because it's expired";
```

Comment:
Ugh. Forgot to hit "Submit".

I think `warn` is too high a log level for this message, especially since the transaction is coming from a peer. `info` or `debug` would be better.
```suggestion
            JLOG(p_journal_.info())
                << "Marking transaction " << stx->getTransactionID()
                << "as BAD because it's expired";
```